### PR TITLE
Use canonical form for lincencing.

### PR DESCRIPTION
**Pull request by elmatou**

### CODE REVIEW

**Comment by jwhitehorn** [comment]
Thank you!

I might have misunderstood your earlier question, but I understand now, and appreciate it.

Not to hijack your PR, but...

I think the copyright notice needs updating in 2 respects;

1. It's no longer 2013
2. I'm no longer the sole contributor

I misread of reading "Copyright (c) 2013, Jason Whitehorn All rights reserved.", it should instead read;

"Copyright (c) 2013-2016, Jason Whitehorn, (however @zsyed91 prefers to be attributed), and contributors All rights reserved."

**Comment by elmatou** [comment]
No problem.
I wouldn't hijack your project, that why I didn't change the content of the notice. I'll update it

**Comment by zsyed91** [comment]
Feel free to add my actual name, Zshawn Syed :smile: :

File: pi_piper.gemspec [hunk 1]
```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.rubyforge_project = "pi_piper"
   s.rubygems_version = "2.0.0"
```
Comment (zsyed91) [inline pi_piper.gemspec]: Instead of using the version directly, do
```ruby
s.rubyforge_project = PiPiper::VERSION
```

**Comment by elmatou** [comment]
I wondered what was de difference beetwen version and rubygems_version :
```
RUBYGEMS_VERSION

The version of RubyGems used to create this gem.

Do not set this, it is set automatically when the gem is packaged.
```
We should remove it. I also have other concernes about the gemspec. Let them be another PR.

**Comment by zsyed91** [comment]
Ah true, that can be another PR. For some reason I thought you added that :smile:.

Looks good and thanks! :+1:

**Pull request merged**